src/data/pokemon/evolution.h

```diff
@@ -70,7 +73,8 @@ const struct Evolution gEvolutionTable[NUM_SPECIES][EVOS_PER_MON] =
     [SPECIES_GRIMER]                = {{EVO_LEVEL, 38, SPECIES_MUK}},
     [SPECIES_SHELLDER]              = {{EVO_ITEM, ITEM_WATER_STONE, SPECIES_CLOYSTER}},
     [SPECIES_GASTLY]                = {{EVO_LEVEL, 25, SPECIES_HAUNTER}},
-    [SPECIES_HAUNTER]               = {{EVO_TRADE, 0, SPECIES_GENGAR}},
+    [SPECIES_HAUNTER]               = {{EVO_TRADE, 0, SPECIES_GENGAR},
+                                       {EVO_ITEM, ITEM_LINKING_CORD, SPECIES_GENGAR}},
```

Comment:
Please add the following Pokémon that would also use the Linking Cord:
- Boldore
- Gurdurr
- Phantump
- Pumpkaboo (and forms)

Reply:
What should be done for Ursaring evo? it says to use Peat Block at full moon night. I tried to make an `EVO_ITEM_USE_NIGHT`
```
case EVO_ITEM_USE_NIGHT: 
        if (gLocalTime.hours >= 0 && gLocalTime.hours < 12 && gEvolutionTable[species][i].param == evolutionItem)                    
              targetSpecies = gEvolutionTable[species][i].targetSpecies;
       break;
```
But it didn't work, the evolution happened regardless of what the time was. Should it just use `EVO_ITEM`?

Reply:
you need to add `RtcCalcLocalTime();` before the if

Reply:
Yeah I did that, still not working, I don't know maybe I didn't set the time right for testing, can you check?

Reply:
Jsyk, you need to test on a savefile that has the wall clock event already completed as that's what handles the initialization of the game's internal clock.

Reply:
Yeah I knew that, the issue was time set didn't fall under 0 - 12 hours. now it's working!